docs/release-instruction.md

```diff
@@ -23,31 +24,27 @@ optimizations, refactoring and API changes. Add a "Behaviour changes" section
 if there are any incompatible changes in default settings or the way commands
 operate.
 
-## Updating ROADMAP
-
 Update ROADMAP.md if necessary.
 
-## Tag the release
-
-Use `vX.Y.Z` tag following the semantic versioning standard.
-
-## Push changes and release tag to Github
+Create a PR with CHANGELOG/ROADMAP changes, review/merge it.
 
-This step should bypass the default PR mechanism to get a correct result (so
-that releasing requires admin privileges for the project). Both the `master`
-branch update and the tag must be pushed simultaneously like this:
+## Create a Github release and a tag
 
-    $ git push origin master v0.70.1
+Use "Draft a new release" button in the "Releases" section. Create a new
+`vX.Y.Z` tag for it following the semantic versioning standard. Put change log
+for this release into the description. Do not attach any binaries at this step.
 
-This is important to ensure proper version in a CI-built binary. Wait for
-CircleCI to build it (or them if we're to have more than one OS or
-architecture), download it and rename to `neo-go-$OS-$ARCH`, at the moment
-that should be `neo-go-linux-amd64`.
+## Add automatically-built binaries
 
-## Make a proper Github release
-
-Edit an automatically-created release on Github, copy things from changelog
-there and attach previously downloaded binary. Make a release.
+New release created at the previous step triggers automatic builds (if not,
+start them manually from the Build Github workflow), so wait for them to
+finish. Then download currently supported binaries (at the time of writing
+that's `neo-go-macos-12-arm64`, `neo-go-ubuntu-20.04-amd64`,
+`neo-go-ubuntu-20.04-arm64` and `neo-go-windows-2022-amd64`), unpack archives and
+rename binaries from them using `neo-go-$GOOS-$GOARCH` scheme, at the moment
+that'd be `neo-go-darwin-arm64`, `neo-go-linux-amd64`, `neo-go-linux-arm64`
+and `neo-go-windows-amd64.exe`. Add these binaries to the previously created
```

Comment:
I've updated our building job at #2727. You don't need to rename binaries anymore. Also, archives names have been changed, although I'm not sure about it and may rollback this change.

Here's the example of resulting archives and binaries: https://github.com/nspcc-dev/neo-go/actions/runs/3194559297.

Reply:
Yeah, this can simplify things a bit.

Reply:
Updated.